frontend/public/locales/fr/translation.json

```diff
@@ -19,8 +19,8 @@
   "video": {
     "notAvailableAnymore": "Cette vidéo n'est plus disponible sur YouTube.",
     "labelShowSettings": "Voir les paramètres de cette video",
+    "insufficientScore": "Le score de cette vidéo est inférieur au seuil de recommendabilité défini par Tournesol.",
```

Comment:
@GresilleSiffle  I opted for "seuil de recommendabilité" instead of "seuil de fiabilité", as I feared it could be confused with the criterion "Fiable et non-trompeur".  Suggestions are welcome.